test/lib/config.test.js

```diff
@@ -44,4 +48,19 @@ describe('config', () => {
         expect(config.get_setting('max-retries')).to.equal('10');
         expect(config.get_setting('retry-backoff-delay')).to.equal('1000');
     });
+
+    it('can pass command line arguments', () => {
+        process.argv = [ 'node', 'my-ingester', '--path', '/tmp/mypath' ];
+        config.parse_options();
+        expect(config.get_setting('path')).to.equal('/tmp/mypath');
+    });
+
+    it('updates command line arguments', () => {
+        process.argv = [ 'node', 'my-ingester', '--path', '/tmp/mypath',
+                         '--no-tgz' ];
+        const options = { argv: [ '--path', './myNewPath' ] };
```

Comment:
I still feel that ingesters shoudl add options directly to the options object and not `argv` but it's too big of a change right now so not a blocker:
```
options = {
  path: '/foo/bar',
}
```